### PR TITLE
Fix more typos

### PR DESCRIPTION
**Pull request by gabrielboliveira**
Fix typos on PUSHNOTIFICATIONS.md and BUILD.md

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -126,9 +126,9 @@ npm run iosEmulator
 For details about various requirements to deploy to a device, refer to the Cordova article: [Getting Started with iOS](http://cordova.apache.org/docs/en/2.5.0/guide_getting-started_ios_index.md.html). Briefly, you need to do the following before deploying:
 
 1. Join the Apple iOS Developer Program.
-1. Generate a certificate http://wiki.genexus.com/commwiki/servlet/hwiki?Create+a+Certificate+Signing+Request+in+a+MAC,
-1. Create a Provisioning Profile within the [iOS Provisioning Portal](https://developer.apple.com/ios/manage/overview/index.action). You can use its Development Provisioning Assistant to create and install the profile and certificate Xcode requires.
-1. Verify that the Code Signing section's Code Signing Identity within the project settings is set to your provisioning profile name.
+2. Generate a certificate http://wiki.genexus.com/commwiki/servlet/hwiki?Create+a+Certificate+Signing+Request+in+a+MAC,
+3. Create a Provisioning Profile within the [iOS Provisioning Portal](https://developer.apple.com/ios/manage/overview/index.action). You can use its Development Provisioning Assistant to create and install the profile and certificate Xcode requires.
```
Comment (gabrielboliveira) [inline BUILD.md]: I know this is not relevant to Github Markdown, but I did it just to be aligned on the file, if someone wants to read it directly

File: BUILD.md [hunk 2]
```diff
@@ -150,5 +150,5 @@ More details: https://github.com/apache/cordova-plugin-whitelist
 To have logs dump in the terminal you can use ```--consolelogs``` options
 
 ```
-npm run ios -- --consolelogs
```
Comment (shprink) [inline BUILD.md]: I think this is correct. We are running that through npm script so the extra `--` will apply the next options to `cordova` in this case.

Comment (gabrielboliveira) [inline BUILD.md]: Understood! Will fix it.

**Comment by gabrielboliveira** [comment]
@shprink fixed the `--` mistake. Hope everything else is ok

**Comment by shprink** [comment]
Thanks very much :)

**Pull request merged**